main/lsp/LocalVarFinder.cc

```diff
@@ -0,0 +1,58 @@
+#include "LocalVarFinder.h"
+#include "ast/ArgParsing.h"
+
+using namespace std;
+
+namespace sorbet::realmain::lsp {
+
+unique_ptr<ast::Assign> LocalVarFinder::postTransformAssign(core::Context ctx, unique_ptr<ast::Assign> assign) {
+    // TODO(jez) We can't guarantee that methodStack is not empty, because we don't run on flattened trees
+    // This means that querys to find the local variables of <static-init> methods will return no results.
+
+    if (methodStack.empty()) {
```

Comment:
can we start with a single entry in `methodStack` that corresponds to staticInitForFile and also add `staticInitForClass` to stack every time we enter classes?
If I understand it right, these ~20 lines will allow to handle variables for `<static-init>`